qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/S3BackupAuthenticationIT.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.backup;
+
+import io.camunda.zeebe.qa.util.testcontainers.MinioContainer;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.camunda.zeebe.test.util.junit.RegressionTest;
+import io.zeebe.containers.ZeebeContainer;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.Network;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+final class S3BackupAuthenticationIT {
```

Comment:
:thought_balloon: It's a bit unfortunate that we have to start the whole zeebe container for testing this. Wondering whether there is any way to do it in unit tests. I found some libraries for this https://github.com/webcompere/system-stubs and https://junit-pioneer.org/docs/environment-variables/. I have never used it, so don't know how useful they are.

Reply:
Yeah that's fair. The AWS SDK also support reading the from properties which are a bit easier to set during runtime I think. I'll see if I can get something working.

Reply:
I think this is sort of possible but I'd like to not do this here:
1.junit-pioneer warns against modifying env variables or system properties. Both have concurrency issues and are not what Java expects.
2. This configuration is passed through a couple of places until it eventually ends up being used by the S3 client. Each of those places can break this so I think it makes sense to test this once, end to end, instead of unit testing a couple of different places.